overviewer_core/src/mc_id.h

```diff
@@ -360,12 +396,16 @@ enum mc_block_id {
     block_jungle_sign = 11404,
     block_acacia_sign = 11405,
     block_dark_oak_sign = 11406,
+    block_crimson_sign = 12505,
+    block_warped_sign = 12506,
```

Comment:
Any particular reason for using high IDs for these and the wall signs? In general we should avoid high IDs because we create an array up to the highest block index, which increases memory usage if this is a new highest ID block.